src/main/java/ch/njol/skript/Skript.java

```diff
@@ -489,6 +489,16 @@ public void onEnable() {
 			classLoadError = e;
 		}
 
+		// Warn about pausing
+		if (Skript.methodExists(Server.class, "getPauseWhenEmptyTime")) {
+			int pauseThreshold = getServer().getPauseWhenEmptyTime();
+			if (pauseThreshold > -1) {
+				Skript.warning("Server pausing is enabled!");
```

Comment:
In my opinion, maybe it should be mentioned somehow that this is a Minecraft feature.
ex:
```suggestion
				Skript.warning("Minecraft server pausing is enabled!");
```
The reason I say this is a lot of server owners dont know of new Minecraft features.
They may not understand this is from Minecraft, and may think "is this a new Skript feature?" "is this from Spigot/Paper?"